README.md

```diff
@@ -46,19 +57,20 @@ Some things to try:
    - Add a breakpoint (e.g. on line 9).
    - Press <kbd>F5</kbd> to launch the app in the container.
    - Once the breakpoint is hit, try hovering over variables (e.g. the app variable on line 7), examining locals, and more.
-   - Continue, then open a local browser and go to `http://localhost:9000` and note you can connect to the server in the container
+   - Continue, then open a local browser and go to `http://localhost:9000` and note you can connect to the server in the container.

```

Comment:
I think we should split this into a Remote Containers set and Codespaces set since you can't just go to localhost:port in a codespace! I'm working on how that might look but feel free to take a stab!

Reply:
Yeah that's a good point! Since users should be notified their port was forwarded, do you think we could mention that notification, and/or maybe just mention the ports panel? Worth including a screenshot of the ports panel and the globe icon next to a forwarded port since I believe that'd be universal across Remote - Containers and Codespaces?